src/translate.rs

```diff
@@ -21,7 +21,14 @@
 //! ```
 //!
 //! `ToGlibPtr`, `FromGlibPtrNone`, `FromGlibPtrFull` and `FromGlibPtrBorrow` work on `gpointer`s
-//! and support different modes of ownership transfer.
+//! and ensure correct ownership of values on the Rust side
```

Comment:
Maybe "ensure correct ownership of values when passing them between Rust and C or the other way around"? It's not just about the Rust side but also the C side